linetools/scripts/lt_radec.py

```diff
@@ -43,10 +43,15 @@ def main(args=None):
     coord = ltu.radec_to_coord(icoord)
 
     # Time to print
+    print('===================================')
     print('J{:s}{:s}'.format(coord.ra.to_string(unit=u.hour,sep='',pad=True),
```

Comment:
I would remove this "===== ..." header, as no every terminal has monospace fonts.

Reply:
changed to a blank line (helps to have that separation)